README

```diff
@@ -1,49 +1,87 @@
 HeapStats
 ===================
 
-HeapStats is Java troubleshooting tool.
-This tool is composed of three programs:
+HeapStats is Java troubleshooting tool.  
+This tool suites of three applications.
 
 * Agent
- * HeapStats JVMTI agent for gathering JavaVM runtime information continuously.
+    * HeapStats JVMTI agent for gathering JavaVM runtime information continuously.
 * Analyzer
- * HeapStats JavaFX GUI analyzer for analysis by various graphs.
+    * HeapStats JavaFX GUI analyzer for analysis by various graphs.
 * CLI
- * HeapStats commandline analyzer.
+    * HeapStats commandline analyzer.
 
+Also two supplemental applications are available:
 
-## Agent ##
+* MBean
+    * MBean for operating HeapStats Agent via JMX
+    * Deploy `heapstats-mbean.jar` and `libheapstats-mbean.so` to the target process
```

Comment:
I think users could not guess how to use. 
How about detailing how to use it or not writing anything as previous three applications?